core/src/main/kotlin/org/evomaster/core/problem/graphql/GqlConst.kt

```diff
@@ -0,0 +1,46 @@
+package org.evomaster.core.problem.graphql
+
+object GqlConst {
+
+    /**
+     * This tag is for the GQL union type. Needed in getValueAsPrintableString to print out things like:
+     *   fieldXName{
+     *        ... on UnionObject1 {
+     *           field
+     *        }
+     *        ... on UnionObjectN {
+     *          field
+     *        }
+     *   }
+     */
+    const val UNION_TAG = "#UNION#"
+    /**
+     * Those tags are for the GQL interface type. Needed in getValueAsPrintableString to print out things like:
+     *     fieldXName{
+     *        field1
+     *        fieldN
+     *        ... on InterfaceObject1 {
+     *           field
+     *        }
+     *        ... on InterfaceObjectN {
+     *          field
+     *        }
+     *     }
+     */
+    const val INTERFACE_BASE_TAG = "#BASE#"
+    const val INTERFACE_TAG = "#INTERFACE#"
+    const val SCALAR_TAG = "scalar"
+    const val OBJECT_TAG = "object"
+    const val UNION_STRING_TAG = "union"
+    const val INTERFACE_STRING_TAG = "interface"
+    const val ENUM_TAG = "enum"
+    const val LIST_TAG = "list"
+    const val INPUT_OBJECT_TAG = "input_object"
+    /**/
```

Comment:
why some `TAG` are in uppercase surrounded by `##` (eg `#BASE#`), whereas others are in lower case (eg `list`).
be consistent.
if those have completely different use in the code, then do not use the same `_TAG` suffix

Reply:
Indeed they are different. those surrounded by `## (eg #BASE#)`  are for a print function and the others `(in lower case(eg list))` are frequently used strings. 
Suffix changed.